web_external/Datasets/UploadDatasetMetadataView.js

```diff
@@ -0,0 +1,195 @@
+isic.views.UploadDatasetMetadataView = isic.View.extend({
```

Comment:
Would `RegisterMetadataView` be descriptive enough? I have trouble parsing the "Dataset" part of `RegisterMetadataDatasetView` when it's not describing the metadata (i.e. "dataset metadata").